tutorials/transfer_learning/README.md

```diff
@@ -0,0 +1,254 @@
+# Transfer learning of vision model with Flux
+
+## Context
+
+This tutorial shows how to perform transfer learning using a pre-trained vision model. In the process, we will also learn how to use a custom data container, a useful feature when dealing with large datasets that cannot fit into memory.
+
+Transfer learning is an common way in which large, compute intensive models can be used in practice. Following their training to perform well on their general trask, they can be subsequently used as a basis to fine-tune only some of their components on smaller, specialized datasets adapted to a specific task.
+
+Self contained Julia code presented in this tutorial is found in ["transfer_learning.jl"](transfer_learning.jl) and can be launched with: 
+
+```
+julia project=@. --threads=8 transfer_learning.jl
+```
+
+## Getting started
+
+In this tutorial, we'll used a pre-trained ResNet18 model to solve a 3-class classification problem: 🐱, 🐶, 🐼.
+
+Data can be accessed from [Kaggle](https://www.kaggle.com/datasets/ashishsaxena2209/animal-image-datasetdog-cat-and-panda).
+
+Following download and unzip, data is expected to live under the following structure: 
+
+```
+- data
+    - animals
+        - cats
+        - dogs
+        - panda
+```
+
+In Julia, the following packages are needed:
+
+```julia
+using Random: shuffle!
+import Base: length, getindex
+using Images
+using Flux
+using Flux: update!
+using DataAugmentation
+using Metalhead
+```
+
+We also define a utility to help manage cpu-gpu conversion:
+
+```julia
+device = Flux.CUDA.functional() ? gpu : cpu
+```
+
+A CUDA enabled GPU is recommended. A modest one with 6GB RAM is sufficient and should run the tutorial in just 5-6 mins. If running on CPU, it may take over 40 mins. 
+
+## Custom DataLoader
```

Comment:
This could be "Custom Dataset" instead

Reply:
I've updated for `Data preparation and loader`